modules/data.land/R/match_species_id.R

```diff
@@ -0,0 +1,65 @@
+#' Match BETY species ID.
+#'
+#' Parses species codes in input data and matches them with the BETY species ID.
+#' 
+#' @param input_codes Character vector of species codes
+#' @param format_name Species code format name (character, length 1)
```

Comment:
To me (character, length 1) implies that this code has to be a single character, which isn't true. I'd either list the valid options right here and/or document the valid options under Details